node.gyp

```diff
@@ -390,7 +391,13 @@
 
       'dependencies': [
         'deps/histogram/histogram.gyp:histogram',
-        'deps/uvwasi/uvwasi.gyp:uvwasi',
+      ],
+      'conditions': [
+        [ 'node_shared_uvwasi=="true"', {
```

Comment:
This is the wrong way round. If building against a shared uvwasi (`'node_shared_uvwasi=="true"'`) then we do not want to build (i.e. depend on) the `uvwasi` code under `deps`.